data/campaigns/emp03.wmf/scripting/texts.lua

```diff
@@ -421,7 +421,10 @@ saledus_5 = {
       _([[But the following objective is even more important:]])
       .. paragraphdivider() ..
       -- TRANSLATORS: Saledus
-      _([[I think I have heard the wind carrying some weird but well known sounds from the north. If I’m not totally wrong, I have heard some Barbarian war drums. So, we should better go and have a look at what is going on up there.]]))
+      _([[I think I have heard the wind carrying some weird but well known sounds from the north. If I’m not totally wrong, I have heard some Barbarian war drums. So, we should better go and have a look at what is going on up there.]])
+      .. paragraphdivider() ..
+      -- TRANSLATORS: Saledus
+      _([[In any case we should be careful with our ports. To defend them against possible enemies we should store some soldiers in each new port build until we can protect it with some military buildings.]]))
```

Comment:
build → built

"store" sounds a bit weird to me, how about "station" or "garrison"? Same for the helptexts

Reply:
well I knew but garrison is what we do in milsites. I am unsure about station. what about send some soldiers to ..